projects/neural_correlations/EXP1-Random/.ipynb_checkpoints/NeuCorr_Exp1-checkpoint.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
I am not familiar with ipython notebook. Is it necessary to check-in the "-checkpoint.ipynb" files? If not, please leave them out.